cfbs/commands.py

```diff
@@ -642,14 +642,27 @@ def install_command(args) -> int:
         destination = "/var/cfengine/masterfiles"
     if len(args) > 0:
         destination = args[0]
+        merge = True
```

Comment:
I think this should be based on an option plus an interactive prompt.

Reply:
Yea, this needs to be an option.